ckanext/ytp/request/templates/request/show.html

```diff
@@ -42,31 +42,38 @@ <h1>{% trans %}Organization membership request{% endtrans %}</h1>
                 <td>{{ membership.request_date }}</td>
               <tr>
                   <th>{% trans %}State{% endtrans %}</th>
-                  <td>{{ membership.state }}</td>
+                  <td>{{ membership.state|capitalize }}</td>
               </tr>
               <tr>
                   <th>{% trans %}Role{% endtrans %}</th>
                   <td>
-
+                    {% if membership.state == 'pending' %}
                       <select id="field-validroles" name="role">
                       {% for option in roles %}
                           <option  value="{{ option.value }}"{% if option.value == membership.role %} selected="selected"{% endif %}>{{ option.text or option.value }}</option>
                       {% endfor %}
                       </select>
-
+                    {% else %}
+                      <span>{{ membership.role|capitalize }}</span>
+                    {% endif %}                      
                   </td>
               </tr>
           </tbody>
-      </table>
-      <div class="form-actions">
-      {% set locale = h.dump_json({'content': _('Are you sure you want approve this request?')}) %}
-     <a id="request_approve_url" href="{{ h.url_for('member_request_approve', mrequest_id=membership.id) }}" class="btn btn-primary  pull-left" data-module="confirm-action" data-module-i18n="{{ locale }}">{{ _('Approve') }}</a>
-
-      {% set locale = h.dump_json({'content': _('Are you sure you want reject this request?')}) %}
-          <a id="request_reject_url" href="{{ h.url_for('member_request_reject', mrequest_id=membership.id) }}" class="btn btn-danger" data-module="confirm-action" data-module-i18n="{{ locale }}">{{ _('Reject') }}</a>
-
-     </div>
-
+      </table>      
+        {% if membership.state == 'pending' %}
+          <div class="form-actions">
+            {% set locale = h.dump_json({'content': _('Are you sure you want approve this request?')}) %}
+            <a id="request_approve_url" href="{{ h.url_for('member_request_approve', mrequest_id=membership.id) }}" class="btn btn-primary  pull-left" data-module="confirm-action" data-module-i18n="{{ locale }}">{{ _('Approve') }}</a>
+            {% set locale = h.dump_json({'content': _('Are you sure you want reject this request?')}) %}
+            <a id="request_reject_url" href="{{ h.url_for('member_request_reject', mrequest_id=membership.id) }}" class="btn btn-danger" data-module="confirm-action" data-module-i18n="{{ locale }}">{{ _('Reject') }}</a>
+          </div>
+        {% else %}
+          <div>
+            <span>This request has already been actioned</span>
```

Comment:
We need to make sure these strings are picked up by our translation system. 

To do this wrap them in {{_('')}}.  The underscore function is shorthand for our internationalisation function. e.g...

```<span>{{_('This request has already been actioned')}}</span>```

Reply:
The other way of doing it is: ```{% trans %}This request has already been actioned{% endtrans %}```